src/prisma/schema.prisma

```diff
@@ -2240,3 +2252,72 @@ model RewardsGithubRepo {
   @@unique([connectionId, repositoryId])
   @@index([connectionId])
 }
+
+model SynapsCredential {
+  id      String  @id @default(uuid()) @db.Uuid
+  apiKey  String
+  secret  String?
+  spaceId String  @unique @db.Uuid
+  space   Space   @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+
+  @@index([spaceId])
+}
+
+enum SynapsUserKycStatus {
+  APPROVED
+  REJECTED
+  RESUBMISSION_REQUIRED
+  SUBMISSION_REQUIRED
+  PENDING_VERIFICATION
+}
+
+model SynapsUserKyc {
+  id        String               @id @default(uuid()) @db.Uuid
+  deletedAt DateTime?
+  sessionId String // Unique identifier for the user KYC session in Synaps
+  status    SynapsUserKycStatus?
+  userId    String               @unique @db.Uuid
+  user      User                 @relation(fields: [userId], references: [id], onDelete: Cascade)
+  spaceId   String               @db.Uuid
+  space     Space                @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+
+  @@unique([userId, spaceId])
+  @@index([userId, spaceId])
+}
+
+model PersonaCredential {
+  id         String  @id @default(uuid()) @db.Uuid
+  apiKey     String
+  envId      String
+  templateId String
+  secret     String?
+  spaceId    String  @unique @db.Uuid
+  space      Space   @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+
+  @@index([spaceId])
+}
+
+enum PersonaUserKycStatus {
+  created
+  pending
+  completed
+  failed
+  expired
+  needs_review
+  approved
+  declined
+}
+
+model PersonaUserKyc {
+  id        String                @id @default(uuid()) @db.Uuid
+  deletedAt DateTime?
+  inquiryId String // Unique identifier for the user KYC inquiry in Persona
+  status    PersonaUserKycStatus?
+  userId    String                @unique @db.Uuid // Unique identifier for the user KYC named referenceId in Persona
```

Comment:
how is this a unique identifier in Persona if it is the user id in our system?

Reply:
The unique identifier is only in our db. We send userId as "reference-id" in persona, ofc a persona inquiry can have multiple inquiry sessions with the same reference-id. 
I admit that the description wasn't that good. I added a better description:
`// This will be stored also in Persona as reference-id so we can debug each user kyc case`